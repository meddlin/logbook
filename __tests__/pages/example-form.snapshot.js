import { render } from '@testing-library/react';
import ExampleForm from '../../pages/example-form';

it('renders ExampleForm unchanged', () => {
    const { container } = render(<ExampleForm />);

  expect(container).toMatchSnapshot()
})