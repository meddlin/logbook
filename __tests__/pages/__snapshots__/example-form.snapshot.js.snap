// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ExampleForm unchanged 1`] = `
<div>
  <form>
    <label
      for="email"
    >
      Email Address
    </label>
    <input
      id="email"
      name="email"
      type="email"
      value=""
    />
    <button
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
